pytest/lib/mocknet.py

```diff
@@ -834,19 +838,188 @@ def create_genesis_file(validator_node_names,
                                increasing_stakes, num_seats, single_shard)
 
 
+def create_and_upload_genesis_file_from_empty_genesis(
```

Comment:
This function duplicates `extra_genesis_records()`. Would be great to reduce the code duplication.

Reply:
Yeah, but I don't feel very comfortable doing this because I don't have a simple way of testing the other function, so I might break something. There's also the difference that I'm not using a records file. Let me just leave it as is for now - if I get time I'd just simply this function instead of merging the two - this one doesn't need to be so complicated.